code/modules/cm_marines/equipment/kit_boxes.dm

```diff
@@ -231,16 +233,17 @@
 	var/selection = tgui_input_list(user, "Pick your specialist equipment type.", "Specialist Kit Selection", GLOB.available_specialist_kit_boxes)
 	if(!selection || QDELETED(src))
 		return FALSE
-	if(!skillcheckexplicit(user, SKILL_SPEC_WEAPONS, SKILL_SPEC_TRAINED) && !skillcheckexplicit(user, SKILL_SPEC_WEAPONS, SKILL_SPEC_ALL))
-		to_chat(user, SPAN_WARNING("You already unwrapped your [name], give this one to someone else!"))
-		return
+	if(!free_skill)
+		if(!skillcheckexplicit(user, SKILL_SPEC_WEAPONS, SKILL_SPEC_TRAINED) && !skillcheckexplicit(user, SKILL_SPEC_WEAPONS, SKILL_SPEC_ALL))
```

Comment:
I don't understand. Isn't rather the problem that the skill check is negated? Wouldn't you want to prevent people that already have skills from using it instead?

Also all of these checks are already in can_use, it could simply be used again